tests/acceptance/test_wineventlog_processing.py

```diff
@@ -35,27 +37,33 @@ def config_template():
 
 
 @pytest.mark.parametrize(
-    "rules, schema, expected_output",
```

Comment:
tests should be written with focus on readability. with rules hidden in some files somewere in the repo it is very hard to see what is tested. we need to speak. this is the case not only here but we have to find a good and readable solution for this

Reply:
as we have discussed this needs to be refactored when we revise the testdata in general